test/Masa.Contrib.Dispatcher.Events.Tests/AssemblyResolutionTests.cs

```diff
@@ -31,9 +31,10 @@ public void TestAddNullAssembly()
         var services = new ServiceCollection();
         services.AddLogging(loggingBuilder => loggingBuilder.AddConsole());
         services.AddTransient(typeof(IMiddleware<>), typeof(LoggingMiddleware<>));
-        Assert.ThrowsException<ArgumentNullException>(() =>
+        Assert.ThrowsException<ArgumentException>(() =>
```

Comment:
Why not ArgumentNullException?

Reply:
Worry about the assignment of empty sets, there is no clear judgment on null and empty sets

```C#
if (assemblies == null || assemblies.Length == 0)
   throw new ArgumentException(nameof(assemblies));
```